src/Umbraco.Web.UI.Client/src/views/content/create.html

```diff
@@ -5,7 +5,7 @@
     <div ng-if="!loading" class="umb-dialog-body with-footer" ng-cloak>
 
         <div class="umb-pane">
-            <h5 ng-show="selectContentType"><localize key="create_createUnder">Create a page under</localize> {{currentNode.name}}</h5>
+            <h5 ng-show="selectContentType" id="selectCotentType"><localize key="create_createUnder">Create a page under</localize> {{currentNode.name}}</h5>
```

Comment:
Think you're missing an "n" there in `selectCotentType` ;-)

Reply:
Fixed there goes an easy PR #4 for hacktobefest :D